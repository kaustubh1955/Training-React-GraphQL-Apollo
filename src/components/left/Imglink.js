import React from 'react';
import '../../styles/Imglink.css'
// import './CounterControl.css';

const Imglink = (props) => (
        <img className= "gugl" src={props.im}/>
    );

export default Imglink;