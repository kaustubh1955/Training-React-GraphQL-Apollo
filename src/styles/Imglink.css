.buttons{
    left: 60px;
    margin-left: 0px;
  }