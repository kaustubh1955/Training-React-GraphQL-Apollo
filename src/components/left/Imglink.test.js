// Link.react.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import Imglink from './Imglink.js';
import appstore from '../../containers/AppStoreBadge.png';

describe('Imglink', () => {
    it('should render correctly in given mode', () => {
        const component = shallow(<Imglink im={appstore} />);
        expect(component).toMatchSnapshot();
    });
});