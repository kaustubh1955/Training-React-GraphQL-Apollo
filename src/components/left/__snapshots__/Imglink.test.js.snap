// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Imglink should render correctly in given mode 1`] = `
<img
  className="gugl"
  src="AppStoreBadge.png"
/>
`;
